datahub-web-react/src/app/entity/shared/containers/profile/header/PlatformContent/ParentNodesView.tsx

```diff
@@ -34,26 +34,27 @@ export const StyledTooltip = styled(Tooltip)`
     overflow: hidden;
 `;
 
-const GlossaryNodeText = styled(Typography.Text)<{ color: string }>`
-    font-size: 12px;
+const GlossaryNodeText = styled(Typography.Text)<{ color: string; fontSize: number }>`
     line-height: 20px;
+    font-size: ${(props) => props.fontSize};
     color: ${(props) => props.color};
 `;
 
-const GlossaryNodeIcon = styled(FolderOutlined)<{ color: string }>`
+const GlossaryNodeIcon = styled(FolderOutlined)<{ color: string; fontSize: number }>`
     color: ${(props) => props.color};
 
     &&& {
-        font-size: 12px;
+        font-size: ${(props) => props.fontSize};
```

Comment:
so above on line 43 it could be `fontSize?: number`and then down below on 47 it could be: `font-size: ${(props) => props.fontSize ? props.fontSize : 12}px;` same with the other styled component above

Reply:
Fixed and committed!!